frame/vamm/src/lib.rs

```diff
@@ -0,0 +1,332 @@
+//! # VAMM Pallet
+//!
+//! ## Overview
+//!
+//! ![](http://www.plantuml.com/plantuml/svg/NSqz2W91343XtbFe0TpqLWk2zyXcuyv09XdoWzTNB2oi7b_rraZqh26dIrUIshbSpYrpnWt0yRKSFLjj5Unacgova0susvWMk0a_Ej0Fm46D7PwEWu64qRiUrsOL_roce7xFA-l-wHi0)
+//!
+//! TODO
+//!
+//! ### Terminology
+//!
+//! * **VAAM**: Acronym for Virtual Automated Market Maker. (TODO: expand definition)
+//!
+//! TODO
+//!
+//! ### Goals
+//!
+//! TODO
+//!
+//! ### Actors
+//!
+//! TODO
+//!
+//! (clearing house)
+//!
+//! ### Implementations
+//!
+//! The VAMM pallet provides implementations for the following traits:
+//!
+//! - [`Vamm`](composable_traits::vamm::Vamm): Exposes functionality for
+//! creating, managing and deprecating virtual automated market makers.
+//!
+//! ## Interface
+//!
+//! TODO
+//!
+//! ### Extrinsics
+//!
+//! TODO
+//!
+//! not applicable
+//!
+//! ### Implemented Functions
+//!
+//! * [`create`](<Pallet as Vamm>::create)
```

Comment:
Did this work? 👀

Reply:
Unfortunately not..

Not quite sure how I'm supposed to write it in order to `cargo doc` pick the right reference :cry:

Reply:
You can't link directly to the trait implementations

Reply:
you can either link to the traits documentation, or not link to anything at all for the trait implementation

Reply:
for some reason this is not supported with rustdoc

Reply:
I found a workaround for it :) 

Take a look into the code [here](https://github.com/ComposableFi/composable/blob/07cf93c5d24dcd3e3d969939c6bc0a72f2f397c1/frame/vamm/src/lib.rs#L44-L47).

cc/ @kevin-composable , @0xangelo